docs/Makefile

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
+#
+
+# You can set these variables from the command line, and also
+# from the environment for the first two.
+SPHINXOPTS    ?=
+SPHINXBUILD   ?= sphinx-build
+SOURCEDIR     = source
```

Comment:
Can we avoid moving document structure if we set `SOURCEDIR=.`? If not so, changes of the links in `README.md` under root dictory should be considered.